### DISCUSSION TITLE
How to enable hyprland package manager?

### DISCUSSION THREAD

**Discussion by omnigenous** [hyprwm/Hyprland#4318] [Q&A] [answered]
How to install or enable hyprpm?

I was reading https://wiki.hyprland.org/Plugins/Using-Plugins/ and it mentioned that preferred way to install plugins is to use `hyprpm`

I tried `hyprpm` and it returned:
```
bash: hyprpm: command not found
```
`yay -S hyprpm`
```
-> No AUR package found for hyprpm
```

**Comment by vaxerski** [comment, marked as answer]
hyprpm is included in hyprland from 0.34.0 onwards, just like hyprctl.
